lib/src/json_schema/validator.dart

```diff
@@ -321,6 +321,15 @@ class Validator {
     }
   }
 
+  void _validateCustomSetAttributes(JsonSchema schema, Instance instance) {
+    schema.customSetAttributes.forEach((attribute, validator) {
```

Comment:
nit: Rather than using the word "attribute", it would probably make more sense to use the more familiar term "keyword".

Reply:
I changed them in some places, but not everywhere. I'll need to do another pass.